### ISSUE TITLE
[fvt] rflash run against ipmi with unsupported option will change the status of node to updating firmware

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#4229]
xcat version:
Version 2.13.8 (git commit 0a25883b2934d8dc2c83dac191d40d0a9273774d, built Tue Oct 31 07:15:38 EDT 2017)

The man page of rflash against ipmi
```
   OpenPOWER BMC specific (using IPMI):
       rflash noderange [hpm_file_path | -d data_directory] [-c|--check] [--retry=count] [-V]
       rflash noderange --recover bmc_file_path
```

Used a unsupported option ``-l``, the status of node was changed to ``updating firmware``. 

```
[root@c910f03c05k25 xcat]# lsdef -l
Object name: fs3
    ....
    status=booted
    statustime=11-02-2017 03:45:56
    tftpserver=c910f03c05k25
    xcatmaster=c910f03c05k25

[root@c910f03c05k25 xcat]# rflash fs3 -l
Error: The option -l is not supported or invalid update file specified

[root@c910f03c05k25 xcat]# lsdef -l
Object name: fs3
    ......
    status=updating firmware
    statustime=11-02-2017 03:45:56
    tftpserver=c910f03c05k25
    xcatmaster=c910f03c05k25
```

**Comment by gurevichmark** [comment]
Still happens with 2.14.2
Will investigate...

**Comment by gurevichmark** [comment]
Fixed by #5422

**Comment by gurevichmark** [comment]
Fixed
